src-ghc/Pact/Server/PactService.hs

```diff
@@ -22,31 +22,41 @@ import Control.Monad.Reader
 import qualified Data.Map.Strict as M
 
 import Data.Aeson as A
+import Data.Int (Int64)
 import Data.Maybe (fromMaybe)
+import Data.Word (Word32, Word64)
 
+import Pact.Gas
+import Pact.Interpreter
+import Pact.Parse (ParsedDecimal(..))
 import Pact.Types.Command
+import Pact.Types.Gas
+import Pact.Types.Logger
+import Pact.Types.Persistence
 import Pact.Types.RPC
 import Pact.Types.Runtime hiding (PublicKey)
 import Pact.Types.Server
-import Pact.Types.Logger
-import Pact.Gas
-import Pact.Parse (ParsedDecimal(..))
 
-import Pact.Interpreter
 
 initPactService :: CommandConfig -> Loggers -> IO (CommandExecInterface PublicMeta ParsedCode)
 initPactService CommandConfig {..} loggers = do
   let logger = newLogger loggers "PactService"
       klog s = logLog logger "INIT" s
       gasRate = fromMaybe 0 _ccGasRate
       gasModel = constGasModel (fromIntegral gasRate)
-      mkCEI p@PactDbEnv {..} = do
+      chainId = fromMaybe 0 _ccChainId
```

Comment:
Just hardcode these, we'll use a repl function

Reply:
hardcoded